themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -0,0 +1,565 @@
+---
+title: "Startup in a Box"
+layout: challenge/single
+description: |
+    Build and deploy your very own startup landing site to the cloud, complete with everything you need to get started!
+meta_desc: |
+    Build and deploy your very own startup landing site to the cloud, complete with everything you need to get started!
+---
+
+### Welcome to the Pulumi Challenge!
+
+### Startup in a Box
+
+Thinking about turning that side project into a little something more? Follow along to stand up a website for your startup on Amazon S3 with Cloudfront and Checkly, all using Pulumi. When you're done, we'll send you a super limited piece of swag unique to this challenge!
+
+### Prerequisites
+
+In order to complete this challenge, you'll need a couple things set up in advance.
+
+* Pulumi account
```

Comment:
What do you think about linking to `app.pulumi.com/signup` and a link to the CLI install instructions (something like [this](https://github.com/pulumi/workshops/tree/main/fundamentals/python#equipment-tools-and-knowledge))?